packages/umd-bundle/dist/pnc-js-libs.js

```diff
@@ -176,6 +178,4 @@
 
     exports.MessageBus = MessageBus;
 
-    Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
rollup v1.32.1:
https://github.com/rollup/rollup/blob/v1.32.1/docs/999-big-list-of-options.md#outputesmodule

rollup v3.3.0:
https://github.com/rollup/rollup/blob/v3.3.0/docs/999-big-list-of-options.md#outputesmodule